pkgs/by-name/co/collector/package.nix

```
@@ -0,0 +1,44 @@
+{ lib, stdenv, fetchFromGitHub, appstream-glib, desktop-file-utils, libadwaita
+, meson, ninja, pkg-config, wrapGAppsHook4, gtk4, python3, gettext, cmake
+, gobject-introspection }:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "collector";
+  version = "1.0.1";
+
+  src = fetchFromGitHub {
+    owner = "mijorus";
+    repo = "collector";
+    rev = "d98a11c65c7de054cb894aeb4b5e963aeb83c0d8";
+    hash = "sha256-un+PvLAHfosX9jWExepWDbDKev7D9TAu+XfOFm7xOyA=";
+  };
+
+  nativeBuildInputs = [
+    meson
+    pkg-config
+    ninja
+    gettext
+    cmake
+    desktop-file-utils
+    appstream-glib
+    wrapGAppsHook4
+  ];
+
+  buildInputs = [
+    gtk4
+    libadwaita
+    gobject-introspection
+    (python3.withPackages (ps: with ps; [ pillow requests pygobject3 ]))
+  ];
+
```

Since there are no tags in the repo, it might be beneficial to fetch the latest commits all the time and there is a updateScript.

```suggestion
  passthru.updateScript = unstableGitUpdater {
    hardcodeZeroVersion = true;
  };

```